clifford/test/test_dpga.py

```diff
@@ -0,0 +1,188 @@
+
+import numpy as np
+import pytest
+from clifford.dpga import *
+
+
+class TestBasicDPGA:
+    def test_non_orthogonal_metric(self):
+        for i, a in enumerate(wbasis):
+            for j, b in enumerate(wbasis):
+                res = a|b
+                if i != j and i%4 == j%4:
+                    assert res == (0.5) + 0*w1
+                else:
+                    assert res == 0*w1
+                print(a|b, end=' ')
+            print()
```

Comment:
This print is pretty pointless, because you've already asserted what the value is by the time you reach it.